explcheck/src/explcheck-cli.lua

```diff
@@ -76,6 +76,12 @@ local function print_usage()
   print("\t--warnings-are-errors\tProduce a non-zero exit code if any warnings are produced by the analysis.")
 end
 
+local function print_version()
+  print("explcheck (expltools ${DATE}) ${VERSION}")
```

Comment:
I doubt the vanilla Lua supports `"${variable}"` syntax.

```
$ explcheck --version
explcheck (expltools ${DATE}) ${VERSION}
Copyright (c) 2024 Vít Starý Novotný
Licenses: LPPL 1.3 or later, GNU GPL v2 or later
```

Reply:
Thank you for the review. You need to run `l3build tag` to replace these placeholders with the correct values, see also `README.md`, section "Notes for distributors".